app/jobs/nightly_syncs_job.rb

```diff
@@ -84,6 +85,33 @@ def sync_bgs_attorneys
     @slack_report << "*Fatal error in sync_bgs_attorneys:* #{error}"
   end
 
+  # Syncs the decision_mailed status of Legacy Appeals with a decision made
+  def sync_decided_appeals
+    AppealState.where(appeal_type: "LegacyAppeal", decision_mailed: false).each do |appeal_state|
```

Comment:
```suggestion
    AppealState.legacy.where(decision_mailed: false).each do |appeal_state|
```
This would work too! Up to you, but I figured I'd just draw attention to the [scopes](https://github.com/department-of-veterans-affairs/caseflow/blob/1c9b94a00bbfc3028ce4a1583263097803de78e6/app/models/concerns/appeal_state_belongs_to_polymorphic_appeal_concern.rb#L17-L18) we have available on AppealStates.